packages/titanium-dialog/src/titanium-dialog.ts

```diff
@@ -193,6 +192,10 @@ export class TitaniumDialogElement extends LitElement {
       padding: 16px 24px;
     }
 
+    :host([scrolls]:not([disableSmoothScroll])) section {
+      -webkit-overflow-scrolling: touch;
```

Comment:
IOS fixed positions items cannot breakout of -webkit-overflow-scrolling.  I made this prop optionally disable so that if one needs to break free they can.